build.py

```diff
@@ -9,17 +9,29 @@
 def create_parser():
     parser = argparse.ArgumentParser(description="Build PEP documents")
     # alternative builders:
-    parser.add_argument("-l", "--check-links", action="store_true")
-    parser.add_argument("-f", "--build-files", action="store_true")
-    parser.add_argument("-d", "--build-dirs", action="store_true")
+    parser.add_argument("-l", "--check-links", action="store_true",
+                        help='Checks validity of links within PEP sources. '
```

Comment:
```suggestion
                        help='Check validity of links within PEP sources. '
```

Per widespread convention using imperative for command descriptions (see, e.g. `python --help`), and specifically due to it describing what the _command_ does, not the _option_ itself.